### PR TITLE
Remove name-blind sift and phone assessment from timeline

### PR DESCRIPTION
**Pull request by tomlovesgithub**
Removed timeline items
---
## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Visit an exercise you know has a name-blind sift and phone assessment on admin ([example](http://admin-develop.judicialappointments.digital/exercises/ZFF4xU8zNvneljZ8Ynsm/timeline))
- Visit the [corresponding vacancy page on apply](http://apply-develop.judicialappointments.digital/vacancy/ZFF4xU8zNvneljZ8Ynsm/)
- Ensure name-blind sift and Telephone assessment don't show on timeline

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Review by mbrookeswebdev — CHANGES_REQUESTED** [review]
Joy confirmed on the ticket the specific requirements, so this works as intended. Maybe just remove unwanted code rather than uncomment it.

**Review by mbrookeswebdev — APPROVED** [review]

**Comment by HalcyonJAC** [comment]
Hi. The code in the Vue component has been commented out but the code for the unit test has been deleted. I think that either both should be commented our or both deleted...

**Review by HalcyonJAC — CHANGES_REQUESTED** [review]
Hi. The code in the Vue component has been commented out but the code for the unit test has been deleted. I think that either both should be commented our or both deleted...

**Review by mbrookeswebdev — APPROVED** [review]

File: src/helpers/Timeline/exerciseTimeline.js [hunk 1]
```diff
@@ -79,28 +79,6 @@ const exerciseTimeline = (data) => {
...
-      timeline.push(
-        createShortlistingMethod('Name-blind sift', data.nameBlindSiftStartDate, data.nameBlindSiftEndDate)
-      );
-    }
-
-    if (data.shortlistingMethods.includes('telephone-assessment')) {
-      timeline.push(createShortlistingMethod('Telephone assessment', data.telephoneAssessmentStartDate, data.telephoneAssessmentEndDate));
-    }
-
     if (data.shortlistingMethods.includes('situational-judgement-qualifying-test')) {
```
Comment (warrensearle) [inline src/helpers/Timeline/exerciseTimeline.js:82]: @tomlovesgithub I wonder if we could use the same timeline code in Apply and Admin, just ensure that it checks for data existence before displaying timeline events. (would mean we can move it to JAC Kit)
That way the logic for what shows on Apply vs Admin will be controlled in one place (in digital-platform where we decide which data to publish to Vacancies).

**Review by warrensearle — CHANGES_REQUESTED** [review]
@tomlovesgithub I wonder if we could use the same timeline code in Apply and Admin, just ensure that it checks for data existence before displaying timeline events. (would mean we can move it to JAC Kit)
That way the logic for what shows on Apply vs Admin will be controlled in one place (in digital-platform where we decide which data to publish to Vacancies).

File: src/helpers/Timeline/exerciseTimeline.js (hunk 1 above)
Comment (tomlovesgithub) [reply src/helpers/Timeline/exerciseTimeline.js:82]: yes please! a lot of the 'application enhancement' tickets would benefit from this change.

Comment (tomlovesgithub) [reply src/helpers/Timeline/exerciseTimeline.js:82]: There are complications with this however, for example, this ticket where name-blind and telephone assessments are hidden from candidates but show to admins

**Comment by tomlovesgithub** [comment]
I'm very supportive of the idea of moving the timeline to JACKIT however i think we should make a separate issue for that as there are some discrepancies in the code and hurdles in terms displaying certain info for admins/applicants.

**Comment by tomlovesgithub** [comment]
I'm very supportive of the idea of moving the timeline to JACKIT however i think we should make a separate issue for that as there are some discrepancies in the code and hurdles in terms displaying certain info for admins/applicants.

**Review by KateMJAC — COMMENTED** [review]
Tested with a positive result. 

This has been sent to UTG. Once I receive feedback, I will update.

**Review by lloback — APPROVED** [review]
This looks ok. Checked on browser

**Review by HalcyonJAC — APPROVED** [review]
Looks good to me now :)

**Review by KateMJAC — APPROVED** [review]
Positive user testing result.

**Review by warrensearle — APPROVED** [review]

**Comment by andrew-isaac** [comment]
Reopened ticket as there is some issues with this and what a user and candidate experience!

As a user running an exercise around 95% of paper sift exercises are name blind - This is only a few where this is not the case. When a user sets up an exercise they select the "Name Blind Sift" option and input their dates onto a timeline.

The correct order for the timeline of events for a name blind paper sift is:

- Open for applications

- Close for applications

- JAC contacts IAs

- Return date for IAs

- Name blind sift



When this is displayed on the apply side to a candidate it is displayed in the incorrect order with the Name Blind Sift date missing. The list should follow the same order as above and where it has "Name Blind Sift" it should display the month and year for the candidate


o

**Pull request merged**